### PR TITLE
Doc page redesign

### LINKED ISSUE #331: Docs page redesign
Redesign the /docs page according to designs here: https://ibm.ent.box.com/file/293256639198
image

---
### End Goal
**Docs and Guides Table of contents - stage 1 - MAY 2018 UPDATE**
As the content team gives us more to work with, we will "beef up" and iterate on this page. 
Redlines: https://ibm.box.com/s/o9ks9uo90n4dq8qfjny1v61qdsqaetk0
(for some reason, it exported a lot of 'headers' but the main item in these redlines works, it's just at the bottom of the list) 
PNG: https://ibm.box.com/s/t8fk3g8ys3gzmw6v9mfppp30m9jjr4d9
Assets/icons 

- 1x icons: https://ibm.box.com/s/36mntgwcw9lmhn52v0nxuqole1qvdnwr
- 2x icons (for retina - pls use): https://ibm.box.com/s/ajqc8ghdfckpdf5i21s9f0kdipass0zx

### PR DESCRIPTION
**Pull request by wexantis**
#### What was fixed?  (Issue # or description of fix)
Fixes #331
Docs page was redesigned

#### Were the changes tested on
- [x] Firefox (Desktop)
- [x] Safari (Desktop)
- [x] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [x] Dymanic Accessability Plugin (DAP)

### CODE REVIEW

File: src/main/content/_assets/css/doc-header.css [hunk 1]
```diff
@@ -194,7 +193,7 @@
     }
 
     #navbar_responsive_button {
-        background-color: #313653;
+        background-image:linear-gradient(-1deg, #141427 0%, #2c2d4f 100%);
```
Comment (kevinwlau) [inline src/main/content/_assets/css/doc-header.css]: Is there a reason to use `background-image` over `background`?
It seems like both should work the same in this case.

**Comment by kevinwlau** [comment]
Here's the orange interactive guide button mentioned above, which doesn't have a hover/active state
image

**Comment by steven1046** [comment]
We need to update the link for the orange interactive guides button to point to "/guides/?search=Interactive&key=tag"

**Comment by kevinwlau** [comment]
It seems we need to check with design team on the look of the hover/active state for the orange interactive guide button.

**Comment by steven1046** [comment]
@cajaygle @megmulsy do you know what the hover/active states for the interactive button should look like?

**Comment by steven1046** [comment]
I asked Jay, and he said the button colors should be inverted so the background is the same orange and the text is white.

**Comment by wexantis** [comment]
I added the new hover state.  I know that the header blending into the background still needs to be fixed, but I haven't been told what colors to use yet. Is there anything else that needs to be done?

**Comment by steven1046** [comment]
image

I think the buttons are slightly too far apart from each other

**Comment by kinueng** [comment]
Here is the existing orange lightning bolt SVG
https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/img/guide_lightning_bolt.svg

**Comment by wexantis** [comment]
Do all the PNG images need to be changed to SVG images or only the lightning bolt images?

**Pull request merged**